src/Sylius/Bundle/CoreBundle/Installer/Setup/LocaleSetup.php

```diff
@@ -100,4 +106,17 @@ private function getLanguageName(string $code): ?string
             return null;
         }
     }
+
+    private function updateLocaleParameter(string $code): void
+    {
+        $content = file_get_contents($this->localeParameterFilePath);
+
+        if (preg_match('/(?<=parameters:\n\s{4}locale: ).*/', $content) === 0) {
+            throw new \RuntimeException(sprintf('File "%s" does not contain locale parameter.', $this->localeParameterFilePath));
+        }
+
+        $content = preg_replace('/(?<=parameters:\n\s{4}locale: ).*/', $code, $content);
+
+        file_put_contents($this->localeParameterFilePath, $content);
+    }
```

Comment:
Why not `YAML` parser? Yaml files can have different amount of space configured.

Reply:
I couldn't find a way to use yaml parser without overwriting comments in the file :/

Reply:
Let's "extract" the parameters section to a separate `parameters.yaml` file and let's import it in the `services.yaml`. It'll allow leaving comments for beginners, at the same time simplifying the code.